lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -1481,6 +1506,104 @@ class BuildIfGroups : public calyx::FuncOpPartialLoweringPattern {
   }
 };
 
+class BuildParGroups : public calyx::FuncOpPartialLoweringPattern {
+  using FuncOpPartialLoweringPattern::FuncOpPartialLoweringPattern;
+
+  LogicalResult
+  partiallyLowerFuncToComp(FuncOp funcOp,
+                           PatternRewriter &rewriter) const override {
+    WalkResult walkResult = funcOp.walk([&](scf::ParallelOp scfParOp) {
+      if (!scfParOp.getResults().empty())
+        return scfParOp.emitError("Currently don't support reduce operation"),
```

Comment:
(I'm personally not a fan of the comma operator used in this situation, I don't think we gain much from it other than a lack of curly braces I guess)